src/api/account_api.js

```diff
@@ -439,7 +439,7 @@ module.exports = {
             method: 'GET',
             params: {
                 type: 'object',
-                required: ['identity', 'secret', 'endpoint'],
+                required: ['identity', 'secret', 'endpoint', 'endpoint_type'],
```

Comment:
Why we made it a requirement?
I can see that all places in the core are calling it with this field, but have we checked `CheckExternalConnectionAPI` in the operator?

Reply:
I added `endpoint_type` as a required field because if you call this RPC function without it, it will fail.
yes, I checked the operator and verified all the calls there would have `endpoint_type`.

Reply:
👍